src/components/patientsList/item/Param.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { StyleSheet, Text, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import FontForgeIcon from "../../common/FontForgeIcon";
+import { Colors } from "../../../constants/styles";
+import {
+  FONT_REGULAR,
+  FONT_SIZE_16,
+} from "../../../constants/styles/typography";
+
+const Param = ({ style, icon, name, value }) => {
```

Comment:
Nazwa Param trochę niewiele mówi o tym co zawiera komponent, może coś w stylu InterviewInfo? Przyjmowane leki i liczba hospitalizacji to podsumowanie informacji z wywiadu

Reply:
Zgadzam się - do zmiany